examples/gas-burned/src/wasm.rs

```diff
@@ -0,0 +1,21 @@
+use gstd::{msg, prelude::*};
```

Comment:
maybe keep mod wasm inside lib rs for small programs?

Reply:
I prefer a common approach for all.